cmd/marker/bridge-marker-gather

```diff
@@ -0,0 +1,48 @@
+#/bin/bash
```

Comment:
Could we make the output machine readable (without doing some regex fun to get each section)? I think JSON should be easy to do with mere `echo`.